tests/SubsitesReportTest.php

```diff
@@ -1,31 +1,42 @@
 <?php
 /**
+ * @broken, SubSites currently is unavailable on SS4
```

Comment:
Same here, add a `return;` to the top, rather than commenting out the whole file